CommandConsole.cpp

```diff
@@ -23,7 +23,7 @@ HOOK_METHOD(MouseControl, OnRender, () -> void)
 
 bool CommandConsole::RunCommand(CommandGui *commandGui, const std::string& cmd)
 {
-    hs_log_file(cmd.c_str());
+    hs_log_file("%s", cmd.c_str());
```

Comment:
Not sure how to address this myself, reverted for now

Reply:
I would honestly worry more about hslua_g_printMessage than RunCommand in that case. RunCommand is going to be something the player typed in whereas hslua_g_printMessage can be anything hidden in a lua script.

Reply:
Actually, hs_log_file("%s", cmd.c_str()) would be safe and hs_log_file(cmd.c_str()) would be vulnerable. Though the player would be hacking their own game here since they're the one putting in commands.